templating/embedding_controllers.rst

```diff
@@ -98,4 +98,36 @@ string syntax for controllers (i.e. **bundle**:**controller**:**action**):
             ) ?>
         </div>
 
+If your controller should be used :doc:`as a service </controller/service>`,
+you can reference it like this instead:
```

Comment:
Ah, and as for my setup, two things:

- I'm trying to follow a ddd-loosely-inspired folder structure, so my AppBundle dir is not a direct child of src, plus not everything is inside AppBundle.
- I'm not very comfortable with the autowiring magic yet, so an error in my service definitions would not come as a surprise.